scripts/superdesk-authoring/authoring.js

```diff
@@ -61,9 +61,9 @@
 
     var DEFAULT_SCHEMA = Object.freeze({
         slugline: {maxlength: 24},
-        headline: {maxlength: 64, order: 1, editor: ['h1', 'h2', 'bold', 'italic', 'underline', 'quote', 'anchor']},
-        abstract: {maxlength: 160, order: 2, editor: ['h1', 'h2', 'bold', 'italic', 'underline', 'quote', 'anchor']},
-        body_html: {order: 5, editor: ['h1', 'h2', 'bold', 'italic', 'underline', 'quote', 'anchor', 'embed', 'picture', 'table']},
```

Comment:
Which idea is fine ?
Let's not discuss only about these lines that belongs to a bigger change (https://github.com/superdesk/superdesk-client-core/pull/168/commits/c8b7b5d2147cc6ec20c7e8a356acdd8dd0be85e9)

What I understand now is:
* the buttons available in the toolbar are now customizable since https://github.com/superdesk/superdesk-client-core/commit/9af018a9b45b03cb1f9a68f5ff5f803ee84430f4
* for each client project, we need now to define these toolbar buttons in `content_types.json`.

Is that right?

But a problem remains: the value of `schema.abstract.editor` in `data-editorformat="schema.abstract.editor"` for instance (https://github.com/superdesk/superdesk-client-core/commit/9af018a9b45b03cb1f9a68f5ff5f803ee84430f4#diff-245b0820e8afbb398822c4050a9cb7aaR36) is not stable. the value changes in the time. What can we do for that ?